oxo.py

```diff
@@ -1,29 +1,159 @@
 class OxoBoard:
+
     def __init__(self):
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+        self.board_rows = 3
+        self.board_columns = 3
+        self.board_array = []
+        # create the board array
+        for x in range(self.board_rows):
+            self.board_array.append([])
+
+            for y in range(self.board_columns):
+                self.board_array[x].append(0)
```

Comment:
Would be slightly more efficient to build the array first and then append it, e.g.:
```
        for x in range(self.board_rows):
            row = []
            for y in range(self.board_columns):
                row.append(0)
            self.board_array.append(row)
```
This cuts down on array accesses through `self.board_array[x]`